UI/db.ts

```diff
@@ -32,6 +32,10 @@ export async function NewIndexedDB(): Promise<Database | Error> {
             (filter: (e: NostrEvent) => boolean) => {
                 return cache.filter(filter);
             },
+            async (id: string) => {
+                console.log("delete", id);
+                await db.events.delete(id);
```

Comment:
data cleaning for users since we can't expect users to remove indexed db manually.